content/blog/scale-kubernetes-clusters-using-hpe-greenlake-terraform-provider.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Upgrade Kubernetes Clusters using HPE GreenLake Terraform Provider
+date: 2023-05-12T04:07:48.888Z
+author: Tanushi Agarwal
+authorimage: /img/photoforblog.jpg
+disable: false
+---
+IaC or Infrastructure as code is an practice of automating the process of managing and provisioning infrastructure through code instead of manual processes. It gives organizations tools to create, manage, and destroy compute resources by statically defining and declaring these resources in codeblocks. It helps to increase operational agility, simplify management, reduce errors, and save cost.
+
+> I﻿n this post, I will explore options to declare and upgrade Kubernetes clusters on HPE GreenLake using the HPE GreenLake Terraform Provider.
```

Comment:
I would suggest to remove the character ">" so the sentence is aligned with the introductory paragraph above.